Documentation/Administrator/Index.rst

```diff
@@ -0,0 +1,62 @@
+﻿.. ==================================================
+.. FOR YOUR INFORMATION
+.. --------------------------------------------------
+.. -*- coding: utf-8 -*- with BOM.
+
+.. include:: ../Includes.txt
+
+
+.. _admin-manual:
+
+Administrator Manual
+====================
+
+.. _admin-installation:
+
+Installation
+------------
+
+You can install the extension either using the TYPO3 extension manager or using composer.
+
+TYPO3 extension manager
+~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Login to the TYPO3 backend
+2. Navigate to ADMIN TOOLS > Extensions
+3. Click on "Get Extensions" at the top of the frame
+4. Update your extension list if it´s older than 24 hours
+5. Search for service_bw2 and click on the install button
+
+Composer
+~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Add service_bw2 to your composer json e.g. using :php:`composer install jweiland/service-bw2`
+
+2. Activate the extension inside the TYPO3 extension manager
+
+.. _admin-configuration:
+
+Configuration
+-------------
+
+You need to edit the extension configuration after the installation. Open the extension manager, search for service_bw2 and click on the settings-icon.
+
+.. figure:: ../Images/AdministratorManual/OpenExtensionConfiguration.jpg
+   :alt: Open extension configuration
```

Comment:
no width?

Reply:
I think it looks better for the short but wide image 😄 . The image shows one row of the extension manager.